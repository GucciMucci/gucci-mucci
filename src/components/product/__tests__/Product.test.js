import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import { mountProduct } from "./helpers/helpers";

describe("Product Component", () => {
  test("Product Component Renders correctly", () => {
    const wrapper = mountProduct();
    wrapper.debug();
    expect(wrapper).toMatchSnapshot();
  });
});
