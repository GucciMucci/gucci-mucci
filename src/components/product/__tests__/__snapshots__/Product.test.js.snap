// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Component Product Component Renders correctly 1`] = `
mountProduct {
  "context": undefined,
  "props": Object {},
  "refs": Object {},
  "state": Object {},
  "updater": Object {
    "enqueueForceUpdate": [Function],
    "enqueueReplaceState": [Function],
    "enqueueSetState": [Function],
    "isMounted": [Function],
  },
}
`;
